ios/RNSScreenStackHeaderConfig.m

```diff
@@ -138,10 +138,21 @@ - (void)updateViewControllerIfNeeded
   }
 }
 
+- (void)layoutNavigtationController
+{
+  UIViewController *vc = _screenView.controller;
+  UINavigationController *navctr = (UINavigationController *)vc.parentViewController;
+  [navctr.view setNeedsLayout];
+  [navctr.view layoutIfNeeded];
```

Comment:
```suggestion

```
It looks like this line is redundant

Reply:
I am not sure when this will be called later. If in some cases it will not be called to late

Reply:
👍